lib/active_shipping/carriers/fedex.rb

```diff
@@ -153,10 +161,129 @@ def find_tracking_info(tracking_number, options = {})
       parse_tracking_response(xml, options)
     end
 
+
+    # Get Shipping labels
+    # Caveats:
+    #  - Only supports singlular packages
```

Comment:
What is the behaviour when multiple packages are passed in then? Since the signature implies _not_ singular package support.